app/javascript/components/user-tags.vue

```diff
@@ -14,7 +14,7 @@
         </span>
       </div>
 
-      <div class="display-tags__tag-list">
+      <div :class="haveTags? 'display-tags__tag-list': 'display-tags__add-btn'">
```

Comment:
creo que la clase con `__add-btn` no está definida

Reply:
ah si, es que no quería que tuviera ninguna clase en verdad, puedo dejarlo como '' ?

Reply:
puedes [definirlo con objeto](https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax) en vez de ternaria:
```
:class="{ 'display-tags__tag-list': haveTags }"
```